spec/lib/vmdb/config_spec.rb

```diff
@@ -31,17 +31,6 @@
     end
   end
 
-  it ".get_file" do
-    server        = EvmSpecHelper.create_guid_miq_server_zone[1]
-    config        = VMDB::Config.new("vmdb")
-    config.config = {:log_depot => {:uri => "smb://server/share", :username => "user", :password => password}}
-    config.save
-
-    expect(VMDB::Config.get_file("vmdb")).to eq(
-      "---\nlog_depot:\n  uri: smb://server/share\n  username: user\n  password: #{enc_pass}\n"
-    )
-  end
-
```

Comment:
You can't kill this test...it's a legit test.  It just happens to use the log_depot section *for* the test.  Just pick another section that has a password...maybe the authentication section.